### PR TITLE
Adds twitter "related" tag

### PR DESCRIPTION
**Pull request by pi0neerpat**
Allows you to specify which accounts you want the user to follow, immediately after they share the tweet. Screenshot:

image

### CODE REVIEW

File: src/TwitterShareButton.ts [hunk 1]
```diff
@@ -5,10 +5,16 @@ import createShareButton from './hocs/createShareButton';
...
+  {
+    title,
+    via,
+    hashtags = [],
+    related = [],
+  }: { title?: string; via?: string; hashtags?: string[]; related?: string[] },
 ) {
   assert(url, 'twitter.url');
   assert(Array.isArray(hashtags), 'twitter.hashtags is not an array');
+  assert(Array.isArray(hashtags), 'twitter.related is not an array');
```
Comment (nygardk) [inline src/TwitterShareButton.ts]: This is supposed to be `Array.isArray(related)`, right?

Comment (pi0neerpat) [reply src/TwitterShareButton.ts]: You are correct. Fixed!

**Comment by Wellusioner** [comment]
image
When I click twitter share button, Nothing happens now.

**Comment by pi0neerpat** [comment]
@Wellusioner can you post your code? 

I just tried this with no issue on latest release `4.1.0`

```js
  <TwitterShareButton
        title="I'm planting trees without spending any money. Check out my grove 🌳"
        via="rtreesdapp"
        related={["rtreesdapp"]}
        url={window.location}
        hashtags={["treefi"]}
      >
        Share your grove
      </TwitterShareButton>
```

**Pull request merged**